tests/scripts/gen_release_notes.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
```

Comment:
Is this help intended to print even on successful execution?

Reply:
No